.storybook/preview.js

```diff
@@ -23,34 +16,33 @@ import builtTokens from '../lib/styles/build/css/design_tokens.css';
 const cssReq = require.context('!!raw-loader!../lib/components', true, /.\.css$/);
 const cssTokenFiles = cssReq
   .keys()
-  .map(filename => ({ filename, content: cssReq(filename).default }));
+  .map((filename) => ({ filename, content: cssReq(filename).default }));
 
-addParameters({
+export const decorators = [
+  (Story) => (
+    <div className={styles.container}>
+      <Story />
+    </div>
+  ),
+];
+
+export const parameters = {
   // configure storybook-design-token
   designToken: {
     files: {
       css: cssTokenFiles,
-    }
+    },
   },
-  
+
   options: {
-    // configure @storybook/theming
-    theme: virtuosoTheme,
-
-    /**
-     * id to select an addon panel
-     * @type {String}
-     */
-    selectedPanel: 'storybook/a11y/panel',
-  }
-});
-
-addDecorator(withA11y);
-
-addDecorator(Story => (
-  <div className={styles.container}>
-    <Story />
-  </div>
-));
-
-addDecorator(withKnobs);
```

Comment:
Are knobs and a11y decorators no longer needed?

Reply:
Let me look back and see. I started with using controls which lead into migrating to es6 which became more of an issue and this may be left over.

Reply:
Ah! This is all handled in the new `manager.js` file.